app/settings/surveys/targeted-surveys/targeted-survey-edit.html

```diff
@@ -19,108 +19,79 @@ <h1 class="mode-context-title">Surveys</h1>
-            <button type="submit" class="button-alpha"  ng-if="!saving_survey" ng-click="saveSurvey()">{{save}}</button>
-            <button type="submit" class="button-alpha"  disabled ng-if="saving_survey">{{saving}}
-              <div class="loading">
-                <div class="line"></div>
-                <div class="line"></div>
-                <div class="line"></div>
-              </div>
-          </button>
-        </div>
     </div>
+
     <div class="main-col">
-        <div class="form-sheet">
+        <div ng-show="targetedSurveysEnabled" class="form-sheet">
             <div class="form-sheet-summary">
                 <h2 class="form-sheet-title">Create a new targeted survey</h2>
                 <p>Send questions via SMS to a list of those phone numbers.</p>
             </div>
-            <div class="stepper" data-accordion-items="stepper-item">
-                <div class="stepper-item active">
+
+            <div class="stepper">
+                <div class="stepper-item" ng-class="{'active': isActiveStep(1), 'disabled': !isActiveStep(1)}">
                     <h2 class="stepper-heading" data-accordion-trigger>
                         <span class="stepper-badge">1</span>
                         Name your survey
                     </h2>
-
-                    <div class="form-field">
-                        <label>Survey name</label>
-                        <input type="text">
-                    </div>                    
-                    <div class="form-field">
-                        <label>Survey description</label>
-                        <textarea></textarea>
+                    <div ng-show="isActiveStep(1)">
+                        <div class="form-field">
+                            <div class="button-group">
+                                <button class="button-alpha" ng-disabled="!isStepComplete(1)" ng-click="completeStepOne()">Continue</button>
+                            </div>
+                        </div>
                     </div>
-                    <div class="form-field">
-                    <div class="button-group">
-                        <button class="button-alpha">Continue</button>
-                        <button class="button">Back</button>
+                </div>
+
+                <div class="stepper-item" ng-class="{'active': isActiveStep(2), 'disabled': !isActiveStep(2)}">
+                    <h2 class="stepper-heading">
+                            <span class="stepper-badge">2</span>
+                                Choose your audience
+                        </h2>
+                       <div ng-show="isActiveStep(2)">
+                        <div class="form-field">
+                            <div class="button-group">
+                                  <button class="button-alpha" ng-disabled="!isStepComplete(2)" ng-click="completeStepTwo()">Continue</button>
```

Comment:
We need to figure out how to disable this button but I think that is easier to do once we have form-elements there